### DISCUSSION TITLE
유니티맵  제페토 빌드잇으로 수정 또는 추가작업 가능여부?

### DISCUSSION THREAD

**Discussion by elimieum** [naverz/zepeto-studio-kor#260] [5. Q&A]
유니티에서 작업한 맵을 제페토월드에 등록 후 제페토 빌드잇으로 추가 작업 또는 수정이 가능한가요?

**Comment by cheolhui-lee** [comment]
안녕하세요.
유니티에서 빌드잇 맵을 로딩하시면 그 순간의 데이터를 기준으로 로딩해서 씬 데이터에 포함을 시키게 됩니다.
그래서 그 이후에 빌드잇으로 작업하시는 내용은 자동으로 유니티에 반영이 되지 않는 점 참고 부탁드립니다.

**Reply by elimieum** [reply]
답변감사합니다^^
